src/website/apps/api/v2/serializers/blogs.py

```
@@ -0,0 +1,98 @@
+            'meta_description',
+            'cover_image_url',
+            'website_category',
+            'order',
+            'content_html',
+            'created',
+            'modified',
+            'is_deleted',
+            'public_identifier',
+            'api_url',
+            'has_slug',
+        ]
```

Ruff (0.15.12)

[warning] Mutable default value for class attribute

(RUF012)